src/Engine/GameAction.java

```diff
@@ -1029,6 +1030,7 @@ public ExplodeAction(Unit unit, Path path, UnitActionType.Explode pType)
       actor = unit;
     }
 
+    @SuppressWarnings("unused") // Unused code has been tested and may be useful at some point
```

Comment:
Eclipse is complaining that `ExplodeAction.getEvents()` is unused?

Reply:
Ah. Never mind. It's because `isLethal` is set to false and never changed anywhere ever.
I normally just wouldn't include it until we had a use case for lethal exploding things, and avoid adding suppression flags.

Reply:
Should I delete the code, then?

Reply:
I probably would, yeah. It'll be simple enough to add again if we ever support that. Until then, it just looks like a YAGNI violation.